src/System.IO.Pipes/src/Resources/Strings.resx

```diff
@@ -136,7 +136,7 @@
     <value>serverName cannot be an empty string.  Use \\\".\\\" for current machine.</value>
   </data>
   <data name="Argument_InvalidHandle" xml:space="preserve">
-    <value>Invalid handle.</value>
+    <value>'handle' has been disposed or is an invalid handle.</value>
```

Comment:
Optionally you could remove quotes from handle - `Handle has been disposed or is invalid.` in all cases

Reply:
done